docs/running-on-mesos.md

```diff
@@ -211,6 +211,14 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
 </tr>
 <tr>
+  <td><code>spark.mesos.executor.cores</code></td>
+  <td>1.0</td>
+  <td>
+    Set the amount of cores to request for running a mesos executor. This setting is only used for Mesos fine-grained mode.
```

Comment:
Lines should be wrapped liked the other properties.  Also, Mesos should be capitalized in both places.